autopush/db.py

```diff
@@ -995,19 +1036,23 @@ class DatabaseManager(object):
 
     router = attrib(default=None)                   # type: Optional[Router]
     message_tables = attrib(default=Factory(list))  # type: List[str]
-    current_msg_month = attrib(init=False)          # type: Optional[str]
+    current_msg_month = attrib(default=None,
```

Comment:
this becomes None w/ allow_table_rotation = False. Instead it should be set to the latest pegged tablename (from get_latest_message_tablename). This should really be where we cache that value.

This shouldn't be None because it propagates everywhere (through all the rust Command/Responses, which right now assumes it's a String vs Option<String>. Also the "current_month" in the router table)